include/tachimawari/joint.hpp

```diff
@@ -35,28 +35,29 @@ class Joint
 
   void set_target_position(float target_position, float speed = 1.0);
   void set_target_position(float present_position, float target_position, float speed);
+  void set_simulator_target_position(float target_position, float speed);
   void set_present_position(float present_position);
   void set_pid_gain(float p, float i, float d);
 
   void interpolate();
 
   uint8_t get_id();
   std::string get_joint_name();
-  int32_t get_position();
-  int32_t get_goal_position();
+  float get_position();
+  float get_goal_position();
```

Comment:
- Make function constant (`foo() const`) for an observable member function which doesn't change states of a class.
- Return constant reference (`const T & foo() const`) for a getter that directly accessing a member variable.